Sources/TuistDependenciesTesting/SwiftPackageManager/Utils/MockSwiftPackageManager.swift

```diff
@@ -0,0 +1,24 @@
+import TSCBasic
+import TuistGraph
+
+@testable import TuistDependencies
+
+public final class MockSwiftPackageManager: SwiftPackageManaging {
+    public init() {}
+
+    var invokedResolve = false
```

Comment:
IMHO this can be also done via the simple function stub but if you reuse this a lot, it's fine to keep, so I'll leave it up to you